app/components/PVutils.test.ts

```diff
@@ -193,6 +193,16 @@ test("getPvValue returns the text value when text is provided", () => {
 });
 
 test("getPvValue decodes and return the base64 value when b64byt is provided", () => {
+  const original =
+    "¡¢£¤¥¦§¨©ª«¬®¯°±²³´µ¶·¸¹º»¼½¾¿ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖ×ØÙÚÛÜÝÞßàáâãäåæçèéêëìíîïðñòóôõö÷øùúûüýþÿ";
```

Comment:
Test feels a bit circular - shouldn't we test with a constant b64 string (like in the other test)?